kotlin-analysis/src/main/kotlin/org/jetbrains/dokka/analysis/resolve/DokkaJsResolverForModuleFactory.kt

```diff
@@ -82,7 +90,7 @@ internal class DokkaJsResolverForModuleFactory(
                 moduleInfo.kotlinLibrary
                     .createKlibPackageFragmentProvider(
                         storageManager = moduleContext.storageManager,
-                        metadataModuleDescriptorFactory = metadataFactories.DefaultDeserializedDescriptorFactory,
```

Comment:
Also a bit unclear where `DefaultDeserializedDescriptorFactory` went - is it now included in `DefaultDescriptorFactory`?